tests/pc/test_po_update.py

```diff
@@ -67,54 +81,54 @@ def test_po_update(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
 
         # Step 1: Remove portchannel members from portchannel
         for member in portchannel_members:
-            duthost.shell("config portchannel member del %s %s" % (portchannel, member))
+            duthost.shell("config portchannel %s member del %s %s" % (namespace_prefix, portchannel, member))
         remove_portchannel_members = True
 
         # Step 2: Remove portchannel ip from portchannel
-        duthost.shell("config interface ip remove %s %s/31" % (portchannel, portchannel_ip))
+        duthost.shell("config interface %s ip remove %s %s/31" % (namespace_prefix, portchannel, portchannel_ip))
         remove_portchannel_ip = True
 
         time.sleep(30)
-        int_facts = duthost.interface_facts()['ansible_facts']
+        int_facts = duthost.interface_facts(namespace=namespace)['ansible_facts']
         pytest_assert(not int_facts['ansible_interface_facts'][portchannel]['link'])
-        pytest_assert(wait_until(120, 10, duthost.check_bgp_statistic, 'ipv4_idle', 1))
+        pytest_assert(wait_until(120, 10, asichost.check_bgp_statistic, 'ipv4_idle', 1))
 
         # Step 3: Create tmp portchannel
-        duthost.shell("config portchannel add %s" % tmp_portchannel)
+        duthost.shell("config portchannel %s add %s" % (namespace_prefix, tmp_portchannel))
         create_tmp_portchannel = True
 
         # Step 4: Add portchannel member to tmp portchannel
         for member in portchannel_members:
-            duthost.shell("config portchannel member add %s %s" % (tmp_portchannel, member))
+            duthost.shell("config portchannel %s member add %s %s" % (namespace_prefix, tmp_portchannel, member))
         add_tmp_portchannel_members = True
 
         # Step 5: Add portchannel ip to tmp portchannel
-        duthost.shell("config interface ip add %s %s/31" % (tmp_portchannel, portchannel_ip))
-        int_facts = duthost.interface_facts()['ansible_facts']
+        duthost.shell("config interface %s ip add %s %s/31" % (namespace_prefix, tmp_portchannel, portchannel_ip))
```

Comment:
may be you can add a new sonic_asic API for config interface if the existing one cannot be reused.